src/Avalonia.Diagnostics/Diagnostics/DevToolsOptions.cs

```diff
@@ -52,5 +52,10 @@ public class DevToolsOptions
         /// Get or set Focus Highlighter <see cref="Brush"/>
         /// </summary>
         public IBrush? FocusHighlighterBrush { get; set; }
+
+        /// <summary>
+        /// Set <see cref="DevToolsViews"/> that show when DevTools launch
+        /// </summary>
+        public DevToolsViews LaunchView { get; init; }
```

Comment:
Why `init`? All other options are mutable.

Reply:
I think DevToolsOptions should be immutable. I proposed it for v 12.